src/otx/recipe/classification/multi_class_cls/efficientnet_v2.yaml

```diff
@@ -62,39 +55,3 @@ overrides:
           init_args:
             mean: [123.675, 116.28, 103.53]
             std: [58.395, 57.12, 57.375]
-      sampler:
-        class_path: otx.algo.samplers.balanced_sampler.BalancedSampler
-
-    val_subset:
-      batch_size: 64
-      to_tv_image: false
-      transforms:
-        - class_path: otx.core.data.transform_libs.torchvision.Resize
-          init_args:
-            scale: $(input_size)
-            is_numpy_to_tvtensor: true
-        - class_path: torchvision.transforms.v2.ToDtype
-          init_args:
-            dtype: ${as_torch_dtype:torch.float32}
-            scale: false
-        - class_path: torchvision.transforms.v2.Normalize
-          init_args:
-            mean: [123.675, 116.28, 103.53]
-            std: [58.395, 57.12, 57.375]
-
-    test_subset:
-      batch_size: 64
-      to_tv_image: false
-      transforms:
-        - class_path: otx.core.data.transform_libs.torchvision.Resize
-          init_args:
-            scale: $(input_size)
-            is_numpy_to_tvtensor: true
-        - class_path: torchvision.transforms.v2.ToDtype
-          init_args:
-            dtype: ${as_torch_dtype:torch.float32}
-            scale: false
-        - class_path: torchvision.transforms.v2.Normalize
-          init_args:
-            mean: [123.675, 116.28, 103.53]
-            std: [58.395, 57.12, 57.375]
```

Comment:
`reset` should be updated.

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/3776/commits/8bbe02cd9116631d163bbd4a80bf3b1f7eae1bea